package VIMO_Fitment;

public class UseOfCustomException {
    public static void main(String[] args) throws CustomException {

        int a = 1/0;
        System.out.println("Exception occured");
    }
}
